syncback/Tiltfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: Python -*-
+
+krsync_path = os.path.join(os.getcwd(), 'krsync.sh')
+
+
+def syncback(name, deploy, paths, target_dir='.'):
+    # name (str) = resource name
+    # k8s_identifier (str) = thing to exec on (pod, deployment/blah, etc)
+    # paths (List[str]) = paths ON CONTAINER to sync. May be files or dirs. May be absolute, or releative to working dir of container
+    # target_dir (str) = dir on local FS to copy files back to (maybe abs or relative to Tiltfile)
+
+    # TODO: if you're rsync-savvy you might want to do the wildcarding manually--
+    #   give an option to turn off automatic +'***'
+    includes = ' '.join(['--include="{}***"'.format(p) for p in paths])
```

Comment:
what does it mean to syncback multiple paths to a single directory? can you give an example?

Reply:
if i have `paths=['package.json', 'yarn.lock', 'migrations/']` and directory `'./letters'` then the resulting files would be `letters/package.json`, `letters/yarn.lock`, `letters/migrations/`. Does that answer your q?